canonical-web-and-design/jaas-dashboard
# Add grouping models by owner

## Done

By using the grouping toggles you can now group the model list by owner or status.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Once your models have loaded, click the "_Group by_: Owner" in the button toggle in the header
- See that your models are now grouped with a separate table for each user

## For follow-up

- Groupings need to persist as refreshing and using the 'back' button do not return to the previous grouping 
  - https://github.com/canonical-web-and-design/jaas-dashboard/issues/260
- The Status column needs the status icons
  - https://github.com/canonical-web-and-design/jaas-dashboard/issues/261

## Details

Fixes #128 

## Screenshots

<img width="1440" alt="Screenshot 2019-12-12 at 18 20 59" src="https://user-images.githubusercontent.com/505570/70738203-2e310900-1d0c-11ea-92da-cc56ce8244c9.png">
